dao/src/main/resources/io/syndesis/dao/deployment.json

```diff
@@ -103,7 +103,7 @@
               "properties": {
                 "filterOld": {
                   "kind": "parameter",
-                  "displayName": "Filter Old",
+                  "displayName": "Filter already polled tweets",
```

Comment:
Or maybe 'Ignore' even instead of 'Filter' ? And 'found' instead of 'polled':

'Ignore previously found Tweets ' ???  ('Ignore Tweets previously found' ?)

Reply:
"Ignore tweets previously found" is good. 
I didn't know what "Filter already polled tweets" meant.